elementor/elementor
# Fix: Elementor edit button missing on Woocommerce New Product Editor [ED-15009]

## PR Checklist



## PR Type
What kind of change does this PR introduce?


## Summary

This PR can be summarized in the following changelog entry:

* “Edit with Elementor” button is missing on product page when using Woocommerce New Product Editor

## Description
An explanation of what is done in this PR

* Added new hook to triger the rendering of the button.

## Test instructions
This PR can be tested by following these steps:

*Install the latest WooCommerce plugin (Version 8.9.2 and up)

*In Elementor > Settings > General > Post Types tick the checkbox next to ‘Products’.
*Activate the new WC feature "Product Editor"
*Go and edit that product - you should see there is an “Edit with Elementor” button in top left corner

## Quality assurance


Fixes #
